src/plots/plot.ts

```diff
@@ -374,16 +353,50 @@ export class Plot extends Component {
   }
 
   /**
+   * Adds a Dataset to the Plot.
+   *
+   * @param {Dataset} dataset
+   * @returns {Plot} The calling Plot.
+   */
+  public addDataset(dataset: Dataset) {
+    return this._addDataset(dataset, true);
+  }
+
+  private _addDataset(dataset: Dataset, dispatchUpdate: boolean) {
```

Comment:
Instead of having a boolean parameter here, can we have this function not do any updating and then have `addDataset()` do the updating manually?